forms-flow-web/src/components/Draft/Edit.js

```diff
@@ -250,7 +250,7 @@ const View = React.memo((props) => {
                 <i className="fa fa-wpforms" aria-hidden="true" /> &nbsp;{" "}
                 {t("Drafts")}/
               </span>{" "}
-              {form.title}
+              {form.title.length > 60 ? form.title.substring(0, 40) + '...' : form.title}
```

Comment:
this is good. but if you make this a function like it will be great 
```
function textTrim(wordLength, text){
return text?.length > wordLength ? text.substring(0, 40) + '...' :  text;
}
```

Reply:
got it . Will update